conda/environments/rmm_dev_cuda10.1.yml

```diff
@@ -19,3 +19,4 @@ dependencies:
 - spdlog>=1.8.5,<1.9
 - cython>=0.29,<0.30
 - gcovr>=5.0
+- cuda-python>=11.5,<12.0
```

Comment:
Does `cuda-python>=11.5` actually work with CUDA 10.1 / 10.2? Does this affect compatibility with platforms like Jetson?

Reply:
No the oldest version it supports is CUDA 11.0

Jetson is currently not supported. If we were to support it, we would already need CUDA 11.0 at a minimum for other reasons (like CUDA Enhanced Compatibility).

Reply:
Okay, thanks! Let’s remove the outdated environment files.

Reply:
Removed!